src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -0,0 +1,222 @@
+        get_latest_dir = lambda path: sorted(Path(ov_dir).iterdir(), key=os.path.getmtime)[0]
+        ov_dir = os.path.join(get_latest_dir(ov_dir))
+        ov_dir = os.path.join(get_latest_dir(ov_dir))
+    return ov_dir
+
+
+
+def parse_arguments():
+    parser = ArgumentParser()
+
+    models_path_help = "Path to directory/ies contains models to dump subgraph (default way is download OMZ). If `--d=False` specify Conformance IRs directory"
+    device_help = "Specify target device. Default value is CPU"
+    ov_help = "OV binary files path"
+    working_dir_help = "Specify working directory"
+    type_help = "Specify conformance type: OP or API"
+    dump_conformance_help = "Set 'True' if you want to create Conformance IRs from custom models. Default value is 'False'"
+
+    parser.add_argument("-m", "--models_path", help=models_path_help, type=str, required=False, default=NO_MODEL_CONSTANT)
+    parser.add_argument("-d", "--device", help= device_help, type=str, required=False, default="CPU")
+    parser.add_argument("-ov", "--ov_path", help=ov_help, type=str, required=False, default=get_ov_path())
+    parser.add_argument("-w", "--working_dir", help=working_dir_help, type=str, required=False, default='temp')
+    parser.add_argument("-t", "--type", help=type_help, type=str, required=False, default="OP")
+    parser.add_argument("-s", "--dump_conformance", help=dump_conformance_help, type=int, required=False, default=1)
+
+    return parser.parse_args()
+
+class Conformance:
+    def __init__(self, device:str, model_path:str, ov_path:str, type:str, working_dir:str):
+        self._device = device
+        self._model_path = model_path
+        self._ov_path = ov_path
+        self._ov_bin_path = get_ov_path(self._ov_path, True)
+        self._working_dir = working_dir
+        if not (type == "OP" or type == "API"):
+            raise Exception("Incorrect conformance type")
+        self._type = type
+
+    def __download_repo(self, https_url: str, version: str):
+        repo_name = https_url[https_url.rfind('/') + 1:len(https_url) - 4]
+        repo_path = os.path.join(self._working_dir, repo_name)
+        if os.path.exists(repo_path):
+            logger.info(f'Repo: {repo_name} exists in {self._working_dir}. Skip the repo download')
+        else:
+            logger.info(f'Started to clone repo: {https_url} to {repo_path}')
+            repo = Repo.clone_from(https_url, repo_path)
+            repo.submodule_update(recursive=True)
+            logger.info(f'Repo {https_url} was cloned sucessful')
+            remote_version = "origin/" + version
+            if remote_version in repo.git.branch('-r').replace(' ', '').split('\n'):
+                branch = repo.git.checkout(version)
+                repo.submodule_update(recursive=True)
+                logger.info(f'Repo {https_url} is on {version}')
+        return repo_path
+
+    def _convert_models(self):
+        omz_tools_path = os.path.join(self._omz_path, "tools", "model_tools")
```

Comment:
Please, take a look on this place. Maybe we need to add check for each variable, because: it might be exists, or not in case of "mo_path", for example.